db_scripts/dump/Dockerfile

```diff
@@ -1,14 +1,5 @@
 FROM git.loc.gov:4567/devops/docker-hub-mirror/amazonlinux:2
-SHELL ["bin/bash", "-c"]
-RUN echo $' \n\
-[pgdg12] \n\
-name=PostgreSQL 12 for RHEL/CentOS 7 - x86_64 \n\
-baseurl=https://download.postgresql.org/pub/repos/yum/12/redhat/rhel-7-x86_64 \n\
-enabled=1 \n\
-gpgcheck=0' >> /etc/yum.repos.d/pgdg.repo
-
-RUN yum makecache && yum -y install postgresql12 \
-    && yum update -y \
+RUN yum update -y && amazon-linux-extras install -y postgresql12 \
```

Comment:
Good call here

Reply:
Thank you!
AWS was slow to add postgres12 to amzon-linux-extras so it was hacked for a bit while waiting for them to catch up. I neglected to check into that before the db upgrade... kinda like not registering the gravity vaccum ahead of time.

Reply:
I hadn't seen that myself – left me thinking it'd be nice to have something like a GitHub bot or pre-commit hook which was basically a general-purpose version of the [eslint-plugin-unicorn expiring TODO comments](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/expiring-todo-comments.md) where you could leave a comment like `# TODO [2021-09-01]: switch to extras package` which would start triggering a warning when the expiration date is past.